flex-checks/src/main/java/org/sonar/flex/checks/DuplicateSwitchCaseImplementationCheck.java

```diff
@@ -22,31 +22,30 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
 import com.sonar.sslr.api.AstNode;
+import com.sonar.sslr.api.AstNodeType;
 import com.sonar.sslr.api.Token;
+import java.text.MessageFormat;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
 import org.sonar.check.Priority;
 import org.sonar.check.Rule;
+import org.sonar.flex.FlexCheck;
 import org.sonar.flex.FlexGrammar;
 import org.sonar.flex.checks.utils.Tags;
 import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
-import org.sonar.squidbridge.checks.SquidCheck;
-import org.sonar.sslr.parser.LexerlessGrammar;
-
-import javax.annotation.Nullable;
-import java.util.List;
-import java.util.Map;
 
 @Rule(
   key = "S1871",
   name = "Two branches in the same conditional structure should not have exactly the same implementation",
   priority = Priority.MAJOR,
   tags = {Tags.DESIGN, Tags.SUSPICIOUS})
 @SqaleConstantRemediation("10min")
-public class DuplicateSwitchCaseImplementationCheck extends SquidCheck<LexerlessGrammar> {
+public class DuplicateSwitchCaseImplementationCheck extends FlexCheck {
 
   @Override
-  public void init() {
-    subscribeTo(FlexGrammar.SWITCH_STATEMENT);
-  }
+  public List<AstNodeType> subscribedTo() {    return Collections.singletonList(FlexGrammar.SWITCH_STATEMENT);  }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this line start at column 5. ([squid:IndentationCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.flex%3Aflex&branch=SONARFLEX-105&issues=AWGPc9yoOl9lQhPgXcMr&open=AWGPc9yoOl9lQhPgXcMr)

[ISSUE_KEY]: AWGPc9yoOl9lQhPgXcMr